powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/entity/Fido2Authenticators.java

```diff
@@ -0,0 +1,259 @@
+
+            // VivoKey
+            Model.of("d7a423ad-3e19-4492-9200-78137dccc136", "VivoKey Apex FIDO2"),
+
+            // Windows Hello
+            Model.of("08987058-cadc-4b81-b6e1-30de50dcbe96", "Windows Hello Hardware Authenticator"),
+            Model.of("6028b017-b1d4-4c02-b4b3-afcdafc96bb2", "Windows Hello Software Authenticator"),
+            Model.of("9ddd1817-af5a-4672-a2b9-3e3dd95000a9", "Windows Hello VBS Hardware Authenticator"),
+
+            // WiSECURE
+            Model.of("504d7149-4e4c-3841-4555-55445a677357", "WiSECURE AuthTron USB FIDO2 Authenticator"),
+
+            // Wultra
+            Model.of("57415531-2e31-2020-3230-323430323237", "Wultra Authenticator 1.1", SignatureType.POSSESSION_KNOWLEDGE),
+
+            // Yubico
+            Model.of("0bb43545-fd2c-4185-87dd-feb0b2916ace", "Security Key NFC by Yubico - Enterprise Edition"),
+            Model.of("149a2021-8ef6-4133-96b8-81f8d5b7f1f5", "Security Key by Yubico with NFC"),
+            Model.of("2fc0579f-8113-47ea-b116-bb5a8db9202a", "YubiKey 5 Series with NFC"),
+            Model.of("6d44ba9b-f6ec-2e49-b930-0c8fe920cb73", "Security Key by Yubico with NFC"),
+            Model.of("73bb0cd4-e502-49b8-9c6f-b59445bf720b", "YubiKey 5 FIPS Series"),
+            Model.of("85203421-48f9-4355-9bc8-8a53846e5083", "YubiKey 5Ci FIPS"),
+            Model.of("a4e9fc6d-4cbe-4758-b8ba-37598bb5bbaa", "Security Key by Yubico with NFC"),
+            Model.of("b92c3f9a-c014-4056-887f-140a2501163b", "Security Key by Yubico"),
+            Model.of("c1f9a0bc-1dd2-404a-b27f-8e29047a43fd", "YubiKey 5 FIPS Series with NFC"),
+            Model.of("c5ef55ff-ad9a-4b9f-b580-adebafe026d0", "YubiKey 5Ci"),
+            Model.of("cb69481e-8ff7-4039-93ec-0a2729a154a8", "YubiKey 5 Series"),
+            Model.of("d8522d9f-575b-4866-88a9-ba99fa02f35b", "YubiKey Bio Series"),
+            Model.of("ee882879-721c-4913-9775-3dfcce97072a", "YubiKey 5 Series"),
+            Model.of("f8a011f3-8c0a-4d15-8006-17111f9edc7d", "Security Key by Yubico"),
+            Model.of("fa2b99dc-9e39-4257-8f92-4a30d23c4118", "YubiKey 5 Series with NFC"),
+            Model.of("34f5766d-1536-4a24-9033-0e294e510fb0", "YubiKey 5 Series CTAP2.1 Preview Expired"),
+            Model.of("83c47309-aabb-4108-8470-8be838b573cb", "YubiKey Bio Series (Enterprise Profile)"),
+
+            // Other authenticators
+            Model.of("ad784498-1902-3f54-b99a-10bb7dbd9588", "Apple MacBook Pro 14-inch, 2021"),
+            Model.of("4ae71336-e44b-39bf-b9d2-752e234818a5", "Apple Passkeys")
+
+    );
+
+    private static final Map<UUID, Model> MODEL_MAP = initializeFromModels(); // prepare a quick to query map with models addressed by AAGUID
+
+    private static Map<UUID, Model> initializeFromModels() {
+        final Map<UUID, Model> vendors = new HashMap<>();
+        for (Model model : MODELS) {
+            vendors.put(model.getAaguid(), model);
+        }
+        return vendors;
```

Comment:
May we return an immutable collection?

Reply:
I still do not understand how streams are more readable than for cycles, but OK. :)

Reply:
It is up to you. But if you prefer _for cycle_ please add at least `return Collections.unmodifiableMap(vendors)`.